xdsl/interpreters/riscv_scf.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any
+
+from xdsl.dialects import riscv_scf
+from xdsl.interpreter import (
+    Interpreter,
+    InterpreterFunctions,
+    ReturnedValues,
+    impl,
+    impl_terminator,
+    register_impls,
+)
+
+
+@register_impls
+class RiscvScfFunctions(InterpreterFunctions):
+    bitwidth = 32
```

Comment:
If this is a constant, it would be better to have capitalized variable name.

Reply:
Good spot. I'm now thinking that maybe the `InterpreterFunctions` superclass should have the bitwidth set, a lot of interpreters subtlely depend on it.

Reply:
#1386